.github/workflows/checks.yml

```diff
@@ -12,7 +12,6 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - '3.9'
           - '3.10'
           - '3.11'
```

Comment:
Is it possible to add 3.12?

Reply:
We're using flake8-django for linting, it doesn't support python 3.12 yet until PR doesn't merged https://github.com/rocioar/flake8-django/pull/134